vault/token_store.go

```diff
@@ -382,16 +411,30 @@ func (ts *TokenStore) paths() []*framework.Path {
 		{
 			Pattern: "lookup",
 
+			DisplayAttrs: &framework.DisplayAttributes{
+				OperationPrefix: operationPrefixToken,
+			},
+
 			Fields: map[string]*framework.FieldSchema{
 				"token": {
 					Type:        framework.TypeString,
 					Description: "Token to lookup (POST request body)",
 				},
 			},
 
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ReadOperation:   ts.handleLookup,
-				logical.UpdateOperation: ts.handleLookup,
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ReadOperation: &framework.PathOperation{
+					Callback: ts.handleLookup,
+					DisplayAttrs: &framework.DisplayAttributes{
+						OperationVerb: "look-up",
+					},
+				},
+				logical.UpdateOperation: &framework.PathOperation{
+					Callback: ts.handleLookup,
+					DisplayAttrs: &framework.DisplayAttributes{
+						OperationVerb: "look-up2",
```

Comment:
Is this the same situation as the `"roles2"` case earlier?

Reply:
Yes, pretty much, we need to disambiguate here to prevent collisions in OperationID's